lib/services/google_play_api.dart

```diff
@@ -0,0 +1,26 @@
+import 'dart:convert';
+import 'package:http/http.dart' as http;
+
+Future<Map<String, dynamic>> getPackageInfo(String pkgName) async {
+  String result = '{}';
+  if (pkgName == 'com.vanced.android.youtube') {
```

Comment:
This breaks design. ReVanced Manager should not make an exception for an app that has nothing to do with ReVanced Manager

Reply:
No worrries, I will remove the exception. I can also try to find the name and icon on the APKMirror if Google Play Store doesn't have the app. So, two sources.